algorand/TmplSig.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, name):
 #            self.map = json.loads(f.read())
 
 
-        self.map = {"name":"lsig.teal","version":6,"source":"","bytecode":"BiABAYEASIAASDEQgQYSRDEZIhJEMRiBABJEMSCAABJEMQGBABJEMQkyAxJEMRUyAxJEIg==",
+        self.map = {"name":"lsig.teal","version":6,"source":"","bytecode":"BiABAYEASIAASDEQgQYSRDEZIhJEMRiBABJEMSCAABJEMQGBABJEMQkyAxJEMRUyAxJEIkM=",
```

Comment:
We cannot change this... it will effect the hashes

Reply:
You are right, I reverted that change and modified the tests accordingly. Just beware that the disparity still exists.